src/main/java/com/github/manolo8/darkbot/backpage/NovaManager.java

```diff
@@ -0,0 +1,205 @@
+        if (jsonObj.get("result").getAsString().equalsIgnoreCase("OK")) {
+            return true;
+        } else {
+            System.out.println("NovaManager: " + response);
+        }
+        return false;
+    }
+
+    /**
+     * Method to buy Nova Agent/Captain
+     * @param amount
+     * @return
+     * @throws IOException
+     */
+
+    public boolean buyNova(int amount) throws IOException {
+        String response = backpageManager.getConnection("ajax/shop.php", Method.POST)
+                .setRawParam("action", "purchase")
+                .setRawParam("category", "special")
+                .setRawParam("itemId", "captain_captain-generic")
+                .setRawParam("amount", amount)
+                .setRawParam("level", "")
+                .setRawParam("selectedName", "")
+                .getContent();
+        this.update(0);
+        JsonObject jsonObj = g.fromJson(response, JsonObject.class);
+        if (jsonObj.get("result").getAsString().equalsIgnoreCase("success")) {
+            return true;
+        } else {
+            System.out.println("NovaManager: " + response);
+        }
+        return false;
+    }
+
+    /**
+     * Method to get Nova Agent/Captain's perk upgrade details
+     * @param agent
+     * @param perk
+     * @return
+     * @throws IOException
+     */
+    public Perk getPerkDetail(Agent agent, Perk perk) throws IOException {
+        String response = backpageManager.getConnection("ajax/captain.php", Method.POST)
+                .setRawParam("command", "getPerkUpgradeDetail")
+                .setRawParam("captainId", agent.getCaptainId())
+                .setRawParam("perkId", perk.getPerkId())
+                .getContent();
+
+        JsonObject jsonObj = g.fromJson(response, JsonObject.class); //Converts the json string to JsonElement without POJO
+        if (jsonObj.get("result").getAsString().equalsIgnoreCase("OK")) {
+            String resp = jsonObj.get("perk").toString();
+            return g.fromJson(resp, Perk.class);
+        } else {
+            System.out.println("NovaManager: " + response);
+        }
+        return null;
+    }
+
+    /**
+     * Method to upgrade Nova Agent/Captain Specific Perk
+     * @param agent
+     * @param perk
+     * @param upgradeLevel
+     * @return
+     * @throws IOException
+     */
+
+    public boolean upgradeAgentPerk(Agent agent, Perk perk, int upgradeLevel) throws IOException {
+        String response = backpageManager.getConnection("ajax/captain.php", Method.POST)
+                .setRawParam("command", "upgradePerk")
+                .setRawParam("captainId", agent.getCaptainId())
+                .setRawParam("perkId", perk.getPerkId())
+                .setRawParam("upgradeLevel", upgradeLevel)
+                .getContent();
+        this.update(0);
+        JsonObject jsonObj = g.fromJson(response, JsonObject.class); //Converts the json string to JsonElement without POJO
```

Comment:
```suggestion
        JsonObject jsonObj = g.fromJson(response, JsonObject.class);
```
those comments are somewhat redundant